DigitalLearningSolutions.Data.Tests/DataServices/TutorialContentDataServiceTests/ObjectiveTests.cs

```diff
@@ -36,5 +36,70 @@ public void GetNonArchivedObjectivesBySectionAndCustomisationId_returns_objectiv
                 }
             }
         }
+
+        [Test]
+        public void GetNonArchivedCcObjectivesBySectionAndCustomisationId_returns_objectives_correctly()
+        {
+            using (new TransactionScope())
+            {
+                connection.Execute("UPDATE Tutorials SET OriginalTutorialID = 1 WHERE TutorialID = 1137");
+                connection.Execute("UPDATE CustomisationTutorials SET DiagStatus = 0 WHERE CusTutID = 704039");

```

Comment:
These should probable have a // Given before them both here and in the next test below